### DISCUSSION TITLE
Problem with collection escapeWhenCastingToString

### DISCUSSION THREAD

**Discussion by Piastra91** [laravel/framework#49242] [Q&A]
Hi all!

I updated my project to laravel 10(from 8) and I see that now we have escapedWhenCastingToString element when I print a collection. Is there a way to hide this as in the past? This create a lot of issues on my code(I have a lot of foreach and ->each)

Thanks!

**Comment by antimech** [comment]
https://github.com/laravel/framework/issues/40420

**Reply by Piastra91** [reply]
I saw this post but nobody tell to him a real solution. This post have 1 year, nobody found a good solution?

**Reply by antimech** [reply]
https://github.com/laravel/framework/issues/40420#issuecomment-1013887832

Like [sajjadgol](https://github.com/sajjadgol) said, type casting `Collection` instance to `array` is incorrect, instead before printing a `Collection`, you should first transform it into `array` by using [toArray() method](https://laravel.com/docs/10.x/collections#method-toarray):
```php
$collection = collect(['Foo', 'Bar', 'Baz']);
dump($collection->toArray());
```
